server/routerlicious/packages/protocol-base/src/protocol.ts

```diff
@@ -29,6 +29,9 @@ export interface IScribeProtocolState {
 	values: [string, ICommittedProposal][];
 }
 
+/**
+ * @deprecated No usage of this function found in any release group. Will be removed
+ */
```

Comment:
If we're already removing it from the public interface, is there any point to keeping it deprecated here? Feels like we should either deprecate it publicly now and remove in the next protocol-base major, or remove it altogether right now if we're super confident there's no usage in partner repos.